.github/CODEOWNERS

```
@@ -17,3 +17,6 @@
 /system-probe_x64/      @DataDog/ebpf-platform @DataDog/agent-platform
 /omnibus-nikos_arm64/   @DataDog/ebpf-platform @DataDog/agent-security @DataDog/agent-platform
 /omnibus-nikos_x64/     @DataDog/ebpf-platform @DataDog/agent-security @DataDog/agent-platform
+
+/windows/               @DataDog/windows-agent
+/build-container.ps1    @DataDog/windows-agent
```

I'd add:

```suggestion
/windows/               @DataDog/windows-agent @DataDog/agent-platform
/build-container.ps1    @DataDog/windows-agent @DataDog/agent-platform
```

so we still get notified if something happens in these images, but that's up to you.